facturx/facturx.py

```diff
@@ -147,3 +147,31 @@ def write_xml(self, path):
         with open(path, 'wb') as f:
             f.write(self.xml_str)
 
+    def __make_dict(self):
+        fields_data = xml_flavor.FIELDS
+        flavor = self.flavor.name
+
+        output_dict = None
```

Comment:
Why dont' you put `output_dict = {}` to init the dict and avoid the confusing `if` stuff below?